test/config.bats

```diff
@@ -65,3 +65,12 @@ function teardown() {
 	[[ "$RES" == *"crio.runtime.runtimes.runc"* ]]
 	[[ "$RES" == *"crio.runtime.runtimes.crun"* ]]
 }
+
+@test "monitor fields should be translated" {
+	# when
+	RES=$("$CRIO_BINARY_PATH" --conmon-cgroup="pod" --conmon="/bin/true" -c "" -d "" config 2>&1)
```

Comment:
This command is failing in the kata CI because the default runtime is set to a value that is not found in the default conf (using CONTAINER_DEFAULT_RUNTIME env var).
Also the error message is not shown in the CI logs because stderr is redirected.

If this test is only about conmon options, maybe we can test for "RUNTIME_TYPE=vm" and skip it in that case? Conmon is not used with runtime_vm.

Reply:
makes sense, thank you!